electron_app/DownloadManager.js

```diff
@@ -0,0 +1,50 @@
+const mtd = require('zeltice-mt-downloader');
+const path = require('path');
+const os = require('os');
+
+const handleFolder = require('./helpers/handleDisk');
+
+let downElements = [];
+
+let downloadManager = (items, access_token) => {
+    let filePath = '';
+
+    let options = {
+        count: 8,
+        headers: {
+            'Authorization': 'Bearer ' + access_token,
+        },
+        onStart: function (meta) {
+            console.log('Download started');
+        },
+        onEnd: function (err, result) {
+            if (err) console.error(err);
+            else {
+                console.log('Download Complete');
+            }
+        }
+    };
+
+    let promiseArr = [];
+    items.forEach(item => {
+        if (item.preserveStructure) {
```

Comment:
With this change, I think you can remove `FilesDatabase.destinationChange` and `FilesDatabase.preserveStructureChange`